roles/lib_openshift/library/oadm_manage_node.py

```diff
@@ -761,6 +762,32 @@ def __init__(self,
         self.verbose = verbose
         self.kubeconfig = Utils.create_tmpfile_copy(kubeconfig)
         self.all_namespaces = all_namespaces
+        self.oc_binary = self._locate_oc_binary()
+
+    @staticmethod
+    def _locate_oc_binary():
+        ''' Find and return oc binary file '''
+        # https://github.com/openshift/openshift-ansible/issues/3410
+        # oc can be in /usr/local/bin in some cases, but that may not
+        # be in $PATH due to ansible/sudo
+        paths = os.environ.get("PATH", os.defpath).split(os.pathsep)
+        extra_paths = ['/usr/local/bin', os.path.expanduser('~/bin')]
+        for path in extra_paths:
+            if path not in paths:
+                paths.append(path)
+
+        oc_binary = 'oc'
+        if six.PY3:
```

Comment:
Please invert this logic. Python 2 should be the 'catch and modify' since it's legacy.